modules/sbt-plugin/src/sbt-test/debug-session/hot-code-reload/build.sbt

```
@@ -0,0 +1,37 @@
+import ch.epfl.scala.debugadapter.sbtplugin.DebugAdapterPlugin
+import ch.epfl.scala.debugadapter.testfmk._
+
+import com.microsoft.java.debug.core.protocol.Events.OutputEvent.Category
+import com.microsoft.java.debug.core.protocol.Types.StackFrame
+
+import java.nio.file.{Files, Path}
+import _root_.io.reactivex.subjects.PublishSubject
+import scala.concurrent.duration._
+import scala.concurrent.ExecutionContext.Implicits.global
+
+val scalaV = "3.3.0"
+
+val checkBreakpoint =
+  inputKey[Unit]("Check the breakpoint of a debug session")
+val hotCodeReplace =
+  inputKey[Unit]("Check the hot code reloading of a debug session")
```

It's not an input key since it does not expect any input from the command line.
```suggestion
  taskKey[Unit]("Check the hot code reloading of a debug session")
```